src/ng-backstretch.js

```diff
@@ -13,7 +13,8 @@ directive('backstretch', ['$window', '$timeout', function($window, $timeout) {
     scope: {
       images: '&backstretchImages',
       duration: '&backstretchDuration',
-      fade: '&backstretchFade'
+      fade: '&backstretchFade',
+      css_transition: '@backstretchCssTransition'
```

Comment:
I think we should make this more appropriately named with a bit more useful functionality.

```javascript
class: '&backstretchActiveClass'
```

Then, rather than setting the class to `active`, we accept a class name as the parameter value. This allows someone to target `.active` differently.